scripts/tf/train.py

```diff
@@ -136,6 +136,9 @@
 # tensorflow device handling
 device, nb_devices = vxm.tf.utils.setup_device(args.gpu)
 
+gpus = tf.config.experimental.list_physical_devices('GPU')
```

Comment:
Not required . You shall be making GradientAccumulation to address this issue.

Reply:
Done